examples/js/loaders/GLTFLoader.js

```diff
@@ -1536,11 +1536,20 @@ THREE.GLTFLoader = ( function () {
 			// The buffer is not interleaved if the stride is the item size in bytes.
 			if ( byteStride && byteStride !== itemBytes ) {
 
-				// Use the full buffer if it's interleaved.
-				array = new TypedArray( bufferView );
+				var ibCacheKey = 'InterleavedBuffer:' + accessorDef.bufferView + ':' + accessorDef.componentType;
+				var ib = parser.cache.get( ibCacheKey );
+
+				if ( !ib ) {
```

Comment:
You need a space after `ib` for the code style, very minor stuff tho.

Reply:
I'm not sure what you mean — like `! ib` instead of `!ib`?

Reply:
```javascript
if ( ! ib ) {
```

Reply:
Thanks, done.